src/analysisd/analysisd.c

```diff
@@ -1051,8 +1140,53 @@ void OS_ReadMSG_analysisd(int m_queue)
 
     mdebug1("Startup completed. Waiting for new messages..");
 
+    unsigned int check_limits_file_interval = Config.eps_limits_file_check;
+
     while (1) {
+
         sleep(1);
+
+        if (limits.enabled) {
+
+            w_mutex_lock(&limit_eps_mutex);
+
+            if (limits.current_cell >= limits.timeframe - 1) {
+
+                limits.current_cell = limits.current_cell > limits.timeframe - 1 ? limits.timeframe - 1 : limits.current_cell;
```

Comment:
Is this line really necessary? I think we will always enter inside this if with `current_cell` storing `limits.timeframe - 1`.